### PR TITLE
[Merged by Bors] - doc: clarify doc of HasProd

### PR DESCRIPTION
**Pull request by riccardobrasca**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/InfiniteSum/Defs.lean [hunk 1]
```diff
@@ -59,7 +59,7 @@ section HasProd
 
 variable [CommMonoid α] [TopologicalSpace α]
 
-/-- Infinite product on a topological monoid
+/-- `HasProd f a` means that the (infinite) product of the `f b` for `b : β` converges to `a`.
```
Comment (faenuccio) [inline Mathlib/Topology/Algebra/InfiniteSum/Defs.lean:62]:
```suggestion
/-- `HasProd f a` means that the (potentially infinite) product of the `f b` for `b : β` converges to `a`.
```

File: Mathlib/Topology/Algebra/InfiniteSum/Defs.lean [hunk 2]
```diff
@@ -70,7 +70,7 @@ this assumption later, for the lemmas where it is relevant.
 These are defined in an identical way to infinite sums (`HasSum`). For example, we say that
 the function `ℕ → ℝ` sending `n` to `1 / 2` has a product of `0`, rather than saying that it does
 not converge as some authors would. -/
-@[to_additive "Infinite sum on a topological monoid
+@[to_additive "`HasSum f a` means that the (infinite) sum of the `f b` for `b : β` converges to `a`.
```
Comment (faenuccio) [inline Mathlib/Topology/Algebra/InfiniteSum/Defs.lean:73]:
```suggestion
@[to_additive "`HasSum f a` means that the (potentially infinite) sum of the `f b` for `b : β` converges to `a`.
```

**Comment by faenuccio** [comment]
Thanks!

**Comment by faenuccio** [comment]
maintainer merge

**Comment by jcommelin** [comment]
Thanks :tada: 

If CI passes, please remove the label `awaiting-CI` and merge this yourself, by adding a comment `bors r+`.

bors d+

**Comment by riccardobrasca** [comment]
bors merge

**Pull request closed without merging**